documentation/tutorials/AnalyzeOutput/modelVSdataVersion2.Rmd

```diff
@@ -0,0 +1,120 @@
+---
+title: "Simple Model-Data Comparisons"
+author: "Tess McCabe"
+date: "4/26/2017"
+output:
+  html_document: default
+  pdf_document: default
+---
+
+
+In this tutorial we will compare model outputs to data outside of the PEcAn web interface. The goal of this is to demonstrate how to perform additional analyses using PEcAn’s outputs. To do this you can download each of the Output files, and then perform the analyses using whatever software you prefer, or you can perform analyses directly on the PEcAn server itself. Here we’ll be analyzing model outputs in R using a browser-based version of RStudio that’s installed on the server
+
+## Starting RStudio Server
+
+1. Open RStudio Server in a new window at **URL/rstudio** 
+ 
+2.	The username is carya and the password is illinois.
+
+3.	To open a new R script click File > New File > R Script
+
+4.	Use the Files browser on the lower right pane to find where your run(s) are located
+
+  + All PEcAn outputs are stored in the output folder. Click on this to open it up.
+
+  + Within the outputs folder, there will be one folder for each workflow execution. For example, click to open the folder PEcAn_99000000001 if that’s your workflow ID
+
+  + A workflow folder will have a few log and settings files (e.g. pecan.xml) and the following subfolders
+
+```  
+run		contains all the inputs for each run
+out		contains all the outputs from each run
+pft		contains the parameter information for each PFT
+```
+
+Within both the run and out folders there will be one folder for each unique model run, where the folder name is the run ID. Click to open the out folder. For our simple case we only did one run so there should be only one folder (e.g. 99000000001). Click to open this folder.
+
+  + Within this folder you will find, among other things, files of the format <year>.nc. Each of these files contains one year of model output in the standard PEcAn netCDF format.
+
+## Load libraries and open connection to the database
+
+```{r}
+#Load every Library used
+library(PEcAn.all)
+library(PEcAn.DB)
+require(RPostgreSQL)
+library(PEcAn.assim.batch)
+library(PEcAn.benchmark)
+
+
+# Open up a connection to The Bety Database
+
+settings <-list(database = list(bety = list(host = "psql-pecan.bu.edu", driver = "PostgreSQL", user = "bety", dbname = "bety", password = "bety")))
+bety <- src_postgres(host = settings$database$bety$host, user = settings$database$bety$user, password = settings$database$bety$password, dbname = settings$database$bety$dbname)
```

Comment:
As of v 0.6.0 the dplyr author recommends not using `src_postgres` ... 

Here is an excerpt from the pre-release notes https://blog.rstudio.org/2017/04/13/dplyr-0-6-0-coming-soon/

--- 
DATABASES

Almost all database related code has been moved out of dplyr and into **a new package, dbplyr**. This makes dplyr simpler, and will make it easier to release fixes for bugs that only affect databases.

To install the development version of dbplyr so you can try it out, run:

```r
devtools::install_github("hadley/dbplyr")
```

There’s one major change, as well as a whole heap of bug fixes and minor improvements. It is now no longer necessary to create a remote “src”. Instead you can work directly with the database connection returned by DBI, reflecting the robustness of the DBI ecosystem. Thanks largely to the work of Kirill Muller (funded by the R Consortium) DBI backends are now much more consistent, comprehensive, and easier to use. That means that there’s no longer a need for a layer between you and DBI.

You can continue to use `src_mysql()`, `src_postgres()`, and `src_sqlite()` (which still live in dplyr), but **I recommend a new style that makes the connection to DBI more clear**:

```r
con <- DBI::dbConnect(RSQLite::SQLite(), ":memory:")
```